mysite/lti/views.py

```diff
@@ -80,20 +88,29 @@ def lti_redirect(request, course_id=None, unit_id=None):
     |  and/or UserSocialAuth.
     |  Finally login Django user and redirect to Course
 
-    :param course_id: course id from launch url
     :param unit_id: unit id from lunch url
     """
     request_dict = pickle.loads(request.session['LTI_POST'])
```

Comment:
> Is pickle considered the Django best practice for storing a dict as session data?

For this case there is not best practice. However we done review of this part and make improvements in https://github.com/raccoongang/socraticqs2/commit/ab841172748e2c6babfce4b1e9002599a5940750